wesnoth-sota.po

```diff
@@ -1185,14 +1195,14 @@ msgstr "Duch a stín. Ty mě budeš následovat."
 #. [message]: speaker=Ardonna
 #: data/campaigns/Secrets_of_the_Ancients/scenarios/05_Blackwater.cfg:402
 msgid "You’re a ghost... a real ghost! I will indeed follow you. Lead on."
-msgstr "Ty jsi duch... opravdový duch! Budu tě sledovat. Věd mě."
+msgstr "Ty jsi duch... opravdový duch! Budu tě následovat. Veď mě."
 
 #. [message]: speaker=Ardonna
 #: data/campaigns/Secrets_of_the_Ancients/scenarios/05_Blackwater.cfg:428
 msgid ""
 "Blackwater Port is defeated, but we must let none escape. Finish them all."
 msgstr ""
-"Přístav Temných Vod je dobyt, ale nesmíme nechat nikoho uniknout. Zabijte je "
+"Přístav Temných vod je dobyt, ale nesmíme nechat nikoho uniknout. Zabijte je "
```

Comment:
pamatuji si vaši lekci o skloňování; pokud to chcete skloněné, změňte to, ale prosím v jednotném čísle (Temné Vody)
```suggestion
"Přístav Temná Voda je dobyt, ale nesmíme nechat nikoho uniknout. Zabijte je "
```